tests/agent_features/test_asgi_distributed_tracing.py

```diff
@@ -12,80 +12,91 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import copy
 import json
+
 import pytest
-import copy
+from testing_support.asgi_testing import AsgiTest
+from testing_support.fixtures import override_application_settings
+from testing_support.validators.validate_transaction_metrics import (
+    validate_transaction_metrics,
+)
 
 from newrelic.api.application import application_instance
+from newrelic.api.asgi_application import ASGIWebTransaction, asgi_application
 from newrelic.api.background_task import BackgroundTask
 from newrelic.api.transaction import current_transaction
-from newrelic.api.asgi_application import asgi_application, ASGIWebTransaction
 
-from testing_support.asgi_testing import AsgiTest
-from testing_support.fixtures import override_application_settings
-from testing_support.validators.validate_transaction_metrics import validate_transaction_metrics
-
-
-distributed_trace_intrinsics = ['guid', 'traceId', 'priority', 'sampled']
-inbound_payload_intrinsics = ['parent.type', 'parent.app', 'parent.account',
-        'parent.transportType', 'parent.transportDuration']
+distributed_trace_intrinsics = ["traceId", "priority", "sampled"]
+inbound_payload_intrinsics = [
+    "parent.type",
+    "parent.app",
+    "parent.account",
+    "parent.transportType",
+    "parent.transportDuration",
+]
```

Comment:
I think this file should probably be reverted

Reply:
In this case, the distributed tracing intrinsics list goes into an expected absenses attributes list.  Since guid is now DT agnostic, it is expected that this make it to the intrinsics list.